### PR TITLE
Ignore case on the dbsize cmd

### PR DESCRIPTION
**Pull request by jjz921024**
redis commands should be case insensitive

127.0.0.1:36666> DBSIZE SCAN
(error) DBSIZE subcommand only supports scan
127.0.0.1:36666> DBSIZE scan
OK

### CODE REVIEW

File: src/commands/cmd_server.cc [hunk 1]
```diff
@@ -331,7 +331,7 @@ class CommandDBSize : public Commander {
       KeyNumStats stats;
       srv->GetLatestKeyNumStats(ns, &stats);
       *output = redis::Integer(stats.n_key);
-    } else if (args_.size() == 2 && args_[1] == "scan") {
+    } else if (args_.size() == 2 && Util::ToLower(args_[1]) == "scan") {
```
Comment (caipengbo) [inline src/commands/cmd_server.cc:334]: `util::EqualICase` is recommended.

Comment (caipengbo) [inline src/commands/cmd_server.cc:334]:
```suggestion
    } else if (args_.size() == 2 && util::EqualICase(args_[1], "scan")) {
```

Comment (jjz921024) [inline src/commands/cmd_server.cc]: update

File: src/commands/cmd_server.cc [hunk 2]
```diff
@@ -331,7 +331,7 @@ class CommandDBSize : public Commander {
       KeyNumStats stats;
       srv->GetLatestKeyNumStats(ns, &stats);
       *output = redis::Integer(stats.n_key);
-    } else if (args_.size() == 2 && args_[1] == "scan") {
+    } else if (args_.size() == 2 && Util::EqualICase(args_[1], "scan")) {
```
Comment (caipengbo) [inline src/commands/cmd_server.cc]:
```suggestion
    } else if (args_.size() == 2 && util::EqualICase(args_[1], "scan")) {
```

**Review by git-hulk — APPROVED** [review]

**Review by caipengbo — APPROVED** [review]

**Pull request merged**